__tests__/components/Subjects.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Subjects from '../../src/components/Subjects'
+
+const setup = (setupProps = {}) => {
+  const defaultProps = { subjectsType: 'primary' }
+  const props = { ...defaultProps, ...setupProps }
+  const wrapper = shallow(<Subjects {...props} />)
+
+  return {
+    props,
+    wrapper
+  }
+}
+
+describe('Subjects component', () => {
+  test('should render', () => {
+    const { wrapper } = setup()
+
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  test('should use subjectsType prop', () => {
+    const { wrapper } = setup({ subjectsType: 'primary' })
+
+    expect(wrapper.instance().props.subjectsType).toEqual('primary')
```

Comment:
Could this test ever be false?

Reply:
This one was 'test' test. One of my first and I haven't known what tests I should write. I'll remove this one.